app-antivirus/clamav/clamav-1.2.0.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="An open source antivirus engine"
+DESCRIPTION="ClamAV is an open source antivirus engine for detecting  \
+trojans, viruses, malware & other malicious threats."
+HOMEPAGE="https://www.clamav.net"
+COPYRIGHT="2001-2007 Tomasz Kojm
+	2007-2023 Cisco Systems"
+PACKAGER="Luca D'Amico <damico.luca91@live.it>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/Cisco-Talos/clamav/archive/refs/tags/clamav-$portVersion.tar.gz"
+CHECKSUM_SHA256="efe7e234fe29a96bf4da23336e38ea2989ed0be162342993e99a11901524f43f"
+SOURCE_DIR="clamav-clamav-$portVersion"
+PATCHES="haiku-$portVersion.patch"
+ADDITIONAL_FILES="
+	clamd.conf
+	freshclam.conf
+	"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/clamav/clamd.conf keep-old
+	settings/clamav/freshclam.conf keep-old
+	"
+
+PROVIDES="
+	clamav$secondaryArchSuffix = $portVersion
+	cmd:clamav_config$secondaryArchSuffix = $portVersion
+	cmd:clambc$secondaryArchSuffix = $portVersion
+	cmd:clamconf$secondaryArchSuffix = $portVersion
+	cmd:clamd$secondaryArchSuffix = $portVersion
+	cmd:clamdscan$secondaryArchSuffix = $portVersion
+	cmd:clamdtop$secondaryArchSuffix = $portVersion
+	cmd:clamscan$secondaryArchSuffix = $portVersion
+	cmd:clamsubmit$secondaryArchSuffix = $portVersion
+	cmd:freshclam$secondaryArchSuffix = $portVersion
+	cmd:sigtool$secondaryArchSuffix = $portVersion
+	lib:libclamav$secondaryArchSuffix = $portVersion
+	lib:libclamunrar$secondaryArchSuffix = $portVersion
+	lib:libclamunrar_iface$secondaryArchSuffix = $portVersion
+	lib:libfreshclam$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	json_c$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libmspack$secondaryArchSuffix
+	lib:libpcre2_8$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcheck$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libjson_c$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libmspack$secondaryArchSuffix
+	devel:libpcre2_8$secondaryArchSuffix
+	devel:libpthread_stubs$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cargo
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:python3
+	cmd:rustc
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. -DENABLE_MILTER=OFF \
+	-DENABLE_EXTERNAL_MSPACK=ON \
+	-DAPP_CONFIG_DIRECTORY=$settingsDir/clamav \
+	-DDATABASE_DIRECTORY=$settingsDir/clamav/db \
+	-DCMAKE_BUILD_TYPE=Release $cmakeDirArgs .
```

Comment:
One tab extra in front of the lines (indent after the first one).

Reply:
Also add next line (prevents on having to build the whole thing if you want to skip the BUILD part with `haikuporter * -F`)
```diff
+	make -C build
```

Reply:
Like this (put $cmakeDirArgs above in the list too):
```
BUILD()
{
	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release \
		$cmakeDirArgs \
		-DENABLE_MILTER=OFF \
		-DENABLE_EXTERNAL_MSPACK=ON \
		-DAPP_CONFIG_DIRECTORY=$settingsDir/clamav \
		-DDATABASE_DIRECTORY=$settingsDir/clamav/db
	make -C build
}
```

Reply:
Ok, I'm going to edit that part. One thing tho: should I keep "make -C build install" in INSTALL?

Reply:
Yes, "make -C build" only takes part of the "build" part, "make -C build install" does the actual installation.

Reply:
Ok thanks, done ;)